main.tf

```diff
@@ -190,13 +181,7 @@ locals {
       concat([var.origin_bucket], concat([""], aws_s3_bucket.origin.*.id))
     )
   )
-
-  bucket_domain_name = (var.use_regional_s3_endpoint || var.website_enabled) ? format(
-    var.website_enabled ? "%s.s3-website%s%s.amazonaws.com" : "%s.s3%s%s.amazonaws.com",
```

Comment:
@nitrocode This change seems to have broken the "s3-website" functionality. It now wants to change the origin `domain_name` from 
`XXXXX.s3-website.eu-west-1.amazonaws.com` to `XXXXX.s3.eu-west-1.amazonaws.com` which results in an `AccessDenied` when loading the domain.

Reply:
Ah what we want is bucket regional domain name output of the bucket. I'll put in another pr

Reply:
I created this ticket https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn/issues/148 after reviewing your comment. Good catch.